src/args.jl

```diff
@@ -288,13 +288,29 @@ const _series_defaults = KW(
 
 const _plot_defaults = KW(
     :plot_title                  => "",
+    :plot_titlefontsize          => 16,
+    :plot_title_location         => :center,           # also :left or :right
+    :plot_fontfamily             => :match,
+    :plot_titlefontfamily        => :match,
+    :plot_titlefonthalign        => :hcenter,
+    :plot_titlefontvalign        => :vcenter,
+    :plot_titlefontrotation      => 0.0,
+    :plot_titlefontcolor         => :match,
     :background_color            => colorant"white",   # default for all backgrounds,
     :background_color_outside    => :match,            # background outside grid,
     :foreground_color            => :auto,             # default for all foregrounds, and title color,
     :fontfamily                  => "sans-serif",
     :size                        => (600,400),
     :pos                         => (0,0),
-    :window_title                 => "Plots.jl",
+    :window_title                => "Plots.jl",
+    :window_titlefontsize        => 14,
+    :window_title_location       => :center,           # also :left or :right
+    :window_fontfamily           => :match,
+    :window_titlefontfamily      => :match,
+    :window_titlefonthalign      => :hcenter,
+    :window_titlefontvalign      => :vcenter,
+    :window_titlefontrotation    => 0.0,
+    :window_titlefontcolor       => :match,
```

Comment:
Does the window title really have all these attributes? Isn't it just the text appearing in the gui window's title bar? In that case the font, etc. would depend on the OS, right? Or am I missing another use case for `window_title`?

Reply:
You are  right. Thanks for pointing out. I didn't really think here. I will delete these and improve the description of `window_title` instead.